main/src/main/java/cgeo/geocaching/log/LogUtils.java

```diff
@@ -495,4 +498,28 @@ private static ImmutableTriple<ImageResult, List<Image>, Runnable> updateImages(
     private static boolean imagePropertiesEqual(final Image oldImg, final Image newImg, final String newTitle) {
         return Objects.equals(oldImg.title, newTitle) && Objects.equals(oldImg.getDescription(), newImg.getDescription());
     }
+
+    /**
+     * Checks, if the logs are representing the same log-entry.
+     * Log-text can be empty (to deal with field-notes)
+     */
+    public static boolean isMatchingLog(final LogEntry log1, final LogEntry log2) {
```

Comment:
Move method to class `LogEntry`?

Reply:
It's no biggie. Is done like this in other places as well....

But for the books: OOP and DDD is about encapsulating data and related logic together....

Reply:
I guess, I will move the whole logic of updating the found-date to a n own class, then this logic of finding a matching cache will be moved there, because it is kind of a special logic for this case. .

Reply:
Moved to LogEntry for now